module/os/linux/zfs/zfs_ctldir.c

```diff
@@ -590,7 +590,8 @@ struct inode *
 zfsctl_root(znode_t *zp)
 {
 	ASSERT(zfs_has_ctldir(zp));
-	igrab(ZTOZSB(zp)->z_ctldir);
+	/* Shouldn't be first ref, so igrab() cannot return NULL */
```

Comment:
Maybe stronger wording, what do you think about?

```suggestion
	/* Must already be referenced so igrab() cannot return NULL */
```

Reply:
I originally had it worded like this, but I overthank it - if it MUST be referenced then why verify?  But I actually agree. :)